packages/radix-vue/src/Calendar/CalendarNext.vue

```diff
@@ -1,10 +1,15 @@
 <script lang="ts">
 import type { PrimitiveProps } from '@/Primitive'
 import type { CalendarIncrement } from '@/shared/date'
+import type { DateValue } from '@internationalized/date'
 
 export interface CalendarNextProps extends PrimitiveProps {
-/** The calendar unit to go forward */
+/** The calendar unit to go forward 
+* @deprecated Use `nextPage` instead
+*/
   step?: CalendarIncrement
+/** The function to be used for the next page. Overwrites the `nextPage` function set on the `CalendarRoot`. */
+  nextPage?: (placeholder: DateValue) => DateValue
```

Comment:
why do we need `nextPage` if we already provide the props in `CalendarRoot`?

Reply:
Same explanation as for `CalendarPrev`.